src/resources/js/tickets-block.js

```diff
@@ -45,16 +45,46 @@ tribe.tickets.block = {
 		'.tribe-block__tickets__item__quantity__remove, .tribe-block__tickets__item__quantity__add',
 		function( e ) {
 			e.preventDefault();
-			var input  = $( this ).parent().find( 'input[type="number"]' );
-			var add    = $( this ).hasClass( 'tribe-block__tickets__item__quantity__add' );
+			var input = $( this ).parent().find( 'input[type="number"]' );
+			var add = $( this ).hasClass( 'tribe-block__tickets__item__quantity__add' );
+			var step = ( undefined !== Number( input[ 0 ].step ) ) ? Number( input[ 0 ].step ) : 1;
+			var originalValue = Number( input[ 0 ].value );
 
-			// stepUp or stepDown the input according to the button that was clicked
-			add ? input[0].stepUp() : input[0].stepDown();
 
-			// Trigger the on Change for the input as it's not handled via stepUp() || stepDown()
-			input.trigger( 'change' );
+			// stepUp or stepDown the input according to the button that was clicked
+			// handle IE/Edge
+			if ( add ) {
+				var max = ( undefined !== Number( input[ 0 ].max ) ) ? Number( input[ 0 ].max ) : 0;
```

Comment:
same comment here on the `undefined`

Reply:
Also, why are we setting `max` to `0` if it doesn't exist? If the RSVP was unlimited, would this not increase?

Reply:
It's a placeholder for unlimited.
A max of 0 makes no sense - that would mean the ticket is sold out, in which case the input shouldn't show.
We can't arbitrarily set a max - some user will come along that wants to make it higher, so we use 0 as a shortcut - essentially `max(0) == unlimited` which is a variable test vs. checking the input value?